consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -202,7 +202,9 @@ public void run() {
             findRequest.where(and(
                     withValue("configurationName = " + configuration.getConfigurationName()),
                     withValue("whenAvailableDate <= " + ClientConstants.getDateFormat().format(new Date())),
-                    not(withSubfield("jobExecutions", withValue("completedFlag = true")))));
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_SUCCESS'"))),
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_PARTIAL'")))
```

Comment:
I think there is something wrong in the snippet you made (or maybe there is something around lightblue-client that I'm not familiar with). The 'jobExecutions.jobStatus' is mentioned twice with contradictions between each other. I think maybe you would like to put an 'or' instead the second 'and'. I will leave this piece of code as it is right now so we can change when we conclude this thread.